### PR TITLE
baseImage auto complete

### PR DESCRIPTION
**Pull request by aar0nTw**
Test Image tag: `ch15810-ae2d5e1e`

Signed-off-by: Aaron Huang <aaroms9733@gmail.com>
Co-Authored-by: Jack Pan <jackpan@infuseai.io>
Co-Authored-by: Timothy Lee <ctiml@infuseai.io>

### CODE REVIEW

**Comment by popcornylu** [comment]
The image pull secret of the selected image is not filled
https://www.notion.so/infuseai/Spec-Build-Custom-Group-Image-7a95b58133c141b293099b6607853779#660975cd5a2f4333869149a451bd9a20
image

**Comment by aar0nTw** [comment]
updated: `ch15810-f7f25809`

**Comment by aar0nTw** [comment]
new image tag `ch15810-aa28000e`

**Comment by aar0nTw** [comment]
newest `ch15810-a9b89d16`

File: packages/client/src/components/images/baseImageRow.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      if (type === ImageType.ALL && url !== urlForGpu) {
+        this.urlDict[`${displayName} (${scopeType} / CPU)`] = url;
+        this.urlDict[`${displayName} (${scopeType} / GPU)`] = urlForGpu || url;
+        this.secretDict[url] = useImagePullSecret;
+        this.secretDict[urlForGpu] = useImagePullSecret;
+        return [
+          `${displayName} (${scopeType} / CPU)`,
+          `${displayName} (${scopeType} / GPU)`
+        ];
+      } else if (type === ImageType.CPU) {
```
Comment (ctiml) [inline packages/client/src/components/images/baseImageRow.tsx:59]: this should be `ImageType.GPU`, right?

**Comment by ctiml** [comment]
LGTM

**Pull request merged**